server/service.go

```diff
@@ -431,6 +435,11 @@ func (svr *Service) RegisterWorkConn(workConn net.Conn, newMsg *msg.NewWorkConn)
 		xl.Warn("No client control found for run id [%s]", newMsg.RunId)
 		return
 	}
+	// Check auth.
+	if err := svr.authVerifier.VerifyNewWorkConn(newMsg); err != nil {
```

Comment:
The messages to start a new work connection between frpc and frps is:
```
frps   ReqWorkConn ->   frpc
       <- NewWorkConn
      StartWorkConn ->
```

There are no response for `NewWorkConn` to tell frpc that auth failed. Should we send `NewWorkConnResp` if auth failed or add error info in `StartWorkConn`? And then close the work connection here.

Reply:
@fatedier We still need to close the connection on server side too. So we can respond with a response including an error and then close. The same goes for `pong` messages